packages/forms/src/UIForm/UIForm.component.js

```diff
@@ -36,8 +36,8 @@ export class UIFormComponent extends React.Component {
 		this.onActionClick = this.onActionClick.bind(this);
 		// control the tv4 language here.
 		const language = getLanguage(props.t);
-		if (typeof props.language === 'function') {
-			Object.assign(language, props.language);
+		if (props.language != null) {
+			Object.assign(language, ...props.language);
```

Comment:
Just to be sure... Are you sure you really want to spread `props.language`, here ?

Considering : 

```
const a = {aa: 11, bb: 22}
const b = {cc: 33, bb: 99}
```

Then : 

```
Object.assign({}, a, b)
{aa: 11, bb: 99, cc: 33}
```

Whereas :

```
Object.assign({}, a, ...b)
VM9957:1 Uncaught TypeError: undefined is not a function
```

Reply:
(if I'm right, I think that a test is missing 😄)